services/director-v2/src/simcore_service_director_v2/models/domains/comp_tasks.py

```diff
@@ -68,6 +68,64 @@ def to_db_model(self, **exclusion_rules) -> Dict[str, Any]:
             comp_task_dict["state"] = RUNNING_STATE_TO_DB[comp_task_dict["state"]].value
         return comp_task_dict
 
+    @classmethod
+    def from_db(cls, db_row) -> "CompTaskAtDB":
+        # clean defaultValues from schema/outputs/output_key/ if necessary
+        for output_data in db_row.get("schema", {}).get("outputs", {}).values():
```

Comment:
What we are trying to achieve here is to hide/exclude ``schema.outputs.defaultValue`` from parsing. We can do that easily when serializing (``json(exclude=...)``) but not for parsing. A couple of thoughts here:

1. ``row`` argument gets modified in ``.from_db(row)``. Would it be more readable to have explicit ``prune(row);  CompTaskAtDB.from_orm(row)``?
2. you are parsing ``row`` and dropping ``row.schema.outpus.defaultValue`` at the level of ``compTask``.  Would it not be better to add this ``Outputs``class? 
3. what about setting in ``Outputs`` the ``Config.schema_extra=False`` , then it will ignore whatever is not defined in the model

Reply:
used extra = Extra.ignore